var/spack/repos/builtin/packages/openfoam/package.py

```diff
@@ -559,6 +559,18 @@ def configure_trapFpe_off(self):
             filter_file(r'trapFpe\s+\d+\s*;', 'trapFpe 0;',
                         controlDict, backup=False)
 
+    @when('@:2106 %aocc@3.2.0')
```

Comment:
Should this be:
```suggestion
    @when('@:2106 %aocc@3.2.0')
```
```suggestion
    @when('@:2106 %aocc@3.2.0:')
```
I assume the same issue will still affect AOCC 3.3?

Reply:
Thanks @adamjstewart for bringing this up, you are right, this will affect for next releases also.
I have added your suggestion in below commit.